src/test/address.ts

```diff
@@ -2,7 +2,7 @@
 "use strict";
 
 import { Address } from "#erdstall/ledger";
-import PRNG from "./random";
+import PRNG from "#erdstall/test/random";
```

Comment:
if you want, we could also create a second anchor `#test` -> `src/test`, so a shortcut for `#erdstall/test`. But up to you.